README.md

```diff
@@ -1,2 +1,44 @@
 # flow-event-processor
 A processor to monitor events in the flow blockchain
+
+Configuration:
+- FlowNode: MainNet or TestNet
```

Comment:
This work with emulator?

Reply:
It will currently work with the emulator when running from VS Code, but not from docker-compose yet.